spdk-sys/nvme_helper.c

```diff
@@ -1,6 +1,19 @@
 #include "nvme_helper.h"
 
+#include <spdk/bdev_module.h>
+#include <spdk/lib/bdev/bdev_internal.h>
 #include <spdk/nvme_spec.h>
+#include <spdk/thread.h>
+
+uint32_t *
+get_nvme_cmd_cdw10(struct spdk_nvme_cmd *cmd) {
```

Comment:
cool, however AFAIK the verb comes at the end in most of the C code we have today. i.e bdev_open() bdev_close(). as such it should be nvme_cmd_cdw10_get().

Reply:
Fixed.